appender.go

```diff
@@ -562,7 +548,7 @@ func (a *Appender) appendRowArray(args []driver.Value) error {
 		}
 
 		if err := info.typeMatch(reflect.TypeOf(v)); err != nil {
-			return fmt.Errorf("type mismatch for column %d: \n%s \n%s", i, err.Error(), errNote)
+			return fmt.Errorf("type mismatch for column %d: \n%s", i, err.Error())
```

Comment:
```suggestion
			return fmt.Errorf("type mismatch for column %d: \n%s", i + 1, err.Error())
```

I think 1-indexed would be more clear since we're talking about columns and not slice elements in the error message.

Reply:
Fair point! I've updated both error messages (append row column type mismatch and table column type mismatch during appender creation).